spec/fixtures/api/sandbox.html

```diff
@@ -69,8 +69,8 @@
     }, false)
```

Comment:
@tarruda It looks like you calling `popup.close()` for the `should open windows in another domain with cross-scripting disabled` is not closing the opened window, not sure why.